### DISCUSSION TITLE
博士论文引用两个以上的文献如何用逗号隔开

### DISCUSSION THREAD

**Discussion by lonlyDan3** [sjtug/SJTUThesis#1077] [提问] [answered]
中文的博士论文中我连续引用两个文献，\cite{1,2},编译后显示是[1-2]。我希望1和2之间是用逗号隔开。我在setup.tex中的设置为：\usepackage[backend=biber,style=gb7714-2015,gbnamefmt=lowercase,sorting=none,gbpunctin=true]。请问要怎样修改哈。

**Comment by hushidong** [comment, marked as answer]
导言区设置：
`\setcounter{gbrefcompress}{3}`

表示从3篇文献开始压缩。

**Reply by lonlyDan3** [reply]
非常感谢，完美解决了我的问题！
